lib/folio_client/job_status.rb

```diff
@@ -61,7 +59,7 @@ def default_wait_secs
     end
 
     def default_timeout_secs
-      5 * 60
+      10 * 60
```

Comment:
I needed to bump this up to run a test script with a real MARC file containing 1.1K records.